docs/source/vcs.rst

```diff
@@ -270,10 +275,6 @@ More details `here <https://github.com/trustbloc/edge-service/blob/master/docs/v
 
 Try it `here <https://w3c-ccg.github.io/vc-http-api/#/Verifier/verifyCredential>`__.
 
-Requesting a VC
-===============
-
-This is explained in the `Integration Guide for Relying Parties <https://github.com/trustbloc/edge-adapter/blob/master/docs/rp/integration/relying_parties.md#request-end-user-credentials>`__.
```

Comment:
is this link intended to be removed?

Reply:
We have it in adapter page - https://github.com/trustbloc/bloc-docs/blob/master/docs/source/adapters.rst#rp-adapter